.lsifrc.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Any way we could have this in a subdirectory to minimize top-level clutter?

Reply:
Yes, can be in a sub directory. What name would you suggest?

Reply:
scripts/lsif ?

Reply:
Alternatively we can move it to the GitHub action source (e.g. `.github/workflows`)

Reply:
I think moving it to `.github/workflows` would be ideal, not sure whether others agree.

Reply:
Maybe `.github/workflow-resources` in case adding things into the `workflows` folder has its own issues.

Reply:
Will give it a try

Reply:
Add it to `github/workflow-resources` to not tamper with a folder owned by github.